payments-model/src/test/java/com/stripe/android/model/CardPaymentDetailTest.kt

```diff
@@ -0,0 +1,84 @@
+package com.stripe.android.model
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.core.model.CountryCode
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+class CardPaymentDetailTest {
+
+    private val card = ConsumerPaymentDetails.Card(
+        id = "QAAAKIL",
+        last4 = "4242",
+        expiryYear = 2500,
+        expiryMonth = 4,
+        brand = CardBrand.Visa,
+        cvcCheck = CvcCheck.Fail,
+        isDefault = false,
+        billingAddress = ConsumerPaymentDetails.BillingAddress(
+            countryCode = CountryCode.US,
+            postalCode = "42424"
+        )
+    )
+
+    @Test
+    fun `isExpired should be true when expiry date is in the past`() {
+        assertThat(card.copy(expiryYear = 2005).isExpired).isTrue()
+    }
+
+    @Test
+    fun `isExpired should be false when expiry date is in the past`() {
+        assertThat(card.isExpired).isFalse()
+    }
+
+    @Test
+    fun `requiresCardDetailsRecollection should be true when isExpired and cvc needs collection`() {
+        val cvcCheck: CvcCheck = mock()
+        whenever(cvcCheck.requiresRecollection).thenReturn(true)
```

Comment:
Do we need a mock for this? It might be cleaner to just use an actual value, e.g. CvcCheck.Fail

Reply:
Yh, I can do that. I saw that there are multiple cases where the `CvcCheck` enum would require cvc collection, so I used this to cover all the bases.